pkg/dockerregistry/client.go

```diff
@@ -250,7 +254,30 @@ func (c *connection) ImageByTag(namespace, name, tag string) (*Image, error) {
 		return nil, err
 	}
 
-	return repo.getTaggedImage(c, searchTag, tag)
+	image, _, err := repo.getTaggedImage(c, searchTag, tag)
+	return image, err
+}
+
+// ImageManifest returns raw manifest of the specified image within the named Docker image repository
+func (c *connection) ImageManifest(namespace, name, tag string) (string, []byte, error) {
```

Comment:
Now that we're almost completely on v2, this client is less and less useful.  We need to decide when we drop support for importing from v1 registries and then set a timeframe for that.  When we do that, we'll need to remove this type (no action for this PR, but action for @mfojtik)